root/static/scripts/edit/URLCleanup.js

```diff
@@ -1245,7 +1245,7 @@ const CLEANUPS = {
     type: LINK_TYPES.license,
     clean: function (url) {
       url = url.replace(/^(https?:\/\/)?([^\/]+\.)?creativecommons\.org\//, 'https://creativecommons.org/');
-      url = url.replace(/^https:\/\/creativecommons\.org\/(licenses|publicdomain)\/(.+)\/((legalcode|deed)((\.|-)[A-Za-z_]+)?)?/, 'https://creativecommons.org/$1/$2/');
+      url = url.replace(/^https:\/\/creativecommons\.org\/(licenses|publicdomain)\/(.+)\/((legalcode|deed)([.-][A-Za-z_]+)?)?/, 'https://creativecommons.org/$1/$2/');
```

Comment:
Incidentally removed capturing group too.

Reply:
Oh, actually, this one was not used, thus it is fine. The three latest groups could probably even be turned into non-capturing groups with `?:` as in:

```javascript
       url = url.replace(/^https:\/\/creativecommons\.org\/(licenses|publicdomain)\/(.+)\/(?:(?:legalcode|deed)(?:[.-][A-Za-z_]+)?)?/, 'https://creativecommons.org/$1/$2/');
```